test/new-e2e/tests/apm/docker_test.go

```diff
@@ -69,7 +69,7 @@ func TestDockerFakeintakeSuiteUDS(t *testing.T) {
 
 // TestDockerFakeintakeSuiteTCP runs basic Trace Agent tests over the TCP transport
 func TestDockerFakeintakeSuiteTCP(t *testing.T) {
-	options := dockerSuiteOpts(uds, awsdocker.WithAgentOptions(
+	options := dockerSuiteOpts(tcp, awsdocker.WithAgentOptions(
```

Comment:
note: this is not related to the PR goals, but a side fix related to the E2E tests that I was working on.